cherry pick i.MX8 specific commits from kirkstone-next

DominikPoggel opened the pull request.

DominikPoggel committed:
i.MX8MM EVA-MI: adjust to hardware changes

DominikPoggel committed:
i.MX8MM EVA-MI: adjust to harware changes, add ethernet phy

StefanLellwitz committed:
linux-fslc: use battery-backed PCF85363 as default RTC

Signed-off-by: Stefan Lellwitz <lel@iesy.com>

DominikPoggel committed:
u-boot-imx: revert PMIC warm reset to cold

DominikPoggel committed:
i.MX8MM EVA-MI: add PCIe

DominikPoggel committed:
linux-fslc: bump Kernel to 6.1.38

DominikPoggel committed:
i.MX8MM EVA-MI: add audio codec

DominikPoggel committed:
i.MX8MM EVA-MI: rearrange device-tree

Move SPI interfaces from EvalBoard to OSM-SoM

DominikPoggel committed:
i.MX8MM EVA-MI: fix USB hotplugging

StefanLellwitz commented on recipes-kernel/linux/linux-fslc_6.1.31/0011-arm64-dts-iesy-add-hdmi-bridge-dsi.patch:
Do we really wanna have this in kirkstone already, even though it doesn't work yet?

DominikPoggel replied:
As this patch is not included in the linux-fslc_6.1.bbappend, I will remove it from this PR

DominikPoggel committed:
Delete recipes-kernel/linux/linux-fslc_6.1.38/0011-arm64-dts-iesy-add-hdmi-bridge-dsi.patch

Remove patchfile #0011 as it is not included in the bbappend file. It will not be added again for linux-fslc because the dsi-interface is not usable in this specific kernel
